src/Data/Arrays.php

```diff
@@ -0,0 +1,439 @@
+     *
+     * @return bool
+     */
+    public static function prepend($array, $value, $key = null)
+    {
+        if ($key === null) {
+            array_unshift($array, $value);
+        } else {
+            $array = array_merge([$key => $value], $array);
+        }
+
+        return $array;
+    }
+
+    /**
+     * Push an item onto the end of an array.
+     *
+     * @param array $array Dafult array, where value to append.
+     * @param mixed $value Value to be append.
+     * @param mixed $key   Key.
+     *
+     * @since 3.0.0
+     *
+     * @return bool
+     */
+    public static function append($array, $value, $key = null)
+    {
+        return array_merge($array, [$key => $value]);
+    }
+
+    /**
+     * Get the unique elements from array.
+     *
+     * @param array $array Array ot evaulated
+     *
+     * @since 3.0.0
+     *
+     * @return array
+     */
+    public static function unique($array)
+    {
+        $results = [];
+        if (self::isMulti($array)) {
+            $array = self::multiToAssoc($array);
+            foreach ($array as $key => $value) {
+                $results[] = $value;
+            }
+            return array_unique($array);
+        }
+
+        return array_unique($array);
+        
+    }
+
+    /**
+     * Get a subset of the items from the given array.
+     *
+     * @param array $array Array to be evaulated.
+     * @param mixed $keys  Keys
+     *
+     * @since 3.0.0
+     *
+     * @return array
+     */
+    public function subSetOfArray(array $array, $keys)
+    {
+        return array_intersect_key($array, array_flip((array) $keys));
+    }
+
+    /**
+     * Remove one or many array items from a given array using "operator" notation.
+     *
+     * @param array        $array Array to be evaluated.
+     * @param array|string $keys Keys.
+     *
+     * Note: Adapted from laravel\framework.
+     *
+     * @see https://github.com/laravel/framework/blob/5.8/LICENSE.md
+     * @since 3.0.0
+     *
+     * @return void
+     */
+    public static function forget(&$array, $keys, $opr = null)
+    {
+        $arrOrg = $array;
+        if (is_null($keys)) {
+            return false;
+        }
+
+        $keys = (array) $keys;
+
+        if ($keys === []) {
+            return false;
+        }
+        foreach ($keys as $key) {
+            // if the exact key exists in the top-level, remove it
+            if (array_key_exists($key, $array)) {
+                unset($array[$key]);
+                continue;
+            }
+
+            if (null !== $opr) {
+                $parts = explode($opr, $key);
+
+                // clean up before each pass
+                $array = &$original;
+
+                while (count($parts) > 1) {
+                    $part = array_shift($parts);
+
+                    if (isset($array[$part]) && is_array($array[$part])) {
+                        $array = &$array[$part];
+                    } else {
+                        continue 2;
```

Comment:
I'm not sure that `2` means.

Reply:
I know that. Here is the explanation:

 The continue 2 jumps to the next iteration of the loop at 2 levels back